srcpkgs/mesa/template

```diff
@@ -22,7 +22,7 @@ license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
 changelog="https://docs.mesa3d.org/relnotes/${version}.html"
```

Comment:
```suggestion
changelog="https://docs.mesa3d.org/relnotes.html"
```

Reply:
Commit message matter? Was going to be lazy and use github's UI

Reply:
don't change the commit message, just force push to this PR